### ISSUE TITLE
Смена главной страницы

### ISSUE THREAD

**Issue by nicothin** [nicothin/nesia#27]
@vladimir075 

Макеты: [настольный](https://www.figma.com/file/OMi3vX6CvJMMciuJ7rIvSI/Terra?node-id=3099%3A1531) и [мобильный](https://www.figma.com/file/OMi3vX6CvJMMciuJ7rIvSI/Terra?node-id=3172%3A2555)

Задача: переверстать.

**Comment by vladimir075** [comment]
верно

**Comment by nicothin** [comment]
https://nicothin.pro/nesia/index.html

**Comment by vladimir075** [comment]
Эта страница main page (=homepage) разнится от страницы landing page и у нее нет формы регистрации (см дизайн)

**Comment by vladimir075** [comment]
![04B96EE1-605E-45C7-A7FB-04DD63162620](https://user-images.githubusercontent.com/17883276/180793470-08cdbc5e-71ff-4045-b9fb-78590422395e.png)
В мобилке айфона почвляются какие то синие поля

**Comment by nicothin** [comment]
> Эта страница main page (=homepage) разнится от страницы landing page и у нее нет формы регистрации (см дизайн)

Не улавливаю сути. Нужно что-то допилить?

>В мобилке айфона появляются какие то синие поля

Сымитировать баг не удалось, однако, я сменил метод вставки картинки на более старый (а значит, широкоподдерживаемый), прошу проверить помогло ли это.

**Comment by vladimir075** [comment]
Формы регистрации на main page нет (она только на "Landing page". Но наверное Олег ее сможет просто убрать.
С мобилки вроде исчезли синие поля.

**Comment by nicothin** [comment]
> Формы регистрации на main page нет (она только на "Landing page". Но наверное Олег ее сможет просто убрать. С мобилки вроде исчезли синие поля.

Да, там стандартный "подвал" был с этими демками. На всякий случай, убрал (сделал копию стандартного подвала без этих демок модальных окон).
